ports/stm32/src/Arduino.cpp

```diff
@@ -93,10 +142,29 @@ void pinMode(int pinIndex, int value) {
 }
 }
 
-template <typename T>
-char * to_string(T data) {
+std::string to_string(int data) {	
+	//TODO:
+	return std::string(" ");
+}
+
+std::string to_string(unsigned int data) {	
+	//TODO:
+	return std::string(" ");
+}
+
+std::string to_string(float data) {	
 	//TODO:
-	return new char;
+	return std::string(" ");
+}
+
+std::string to_string(double data) {	
+	//TODO:
+	return std::string(" ");
+}
+
+std::string to_string(long unsigned int data) {	
+	//TODO:
```

Comment:
This should work or not available at all. Should be easily done with sprintf.

Reply:
I did not thought about sprintf. I will change that in next update.